includes/class-sensei-course.php

```diff
@@ -2521,7 +2521,7 @@ public static function course_archive_sorting( $query ) {
 		?>
 
 		<form class="sensei-ordering" name="sensei-course-order" action="<?php echo esc_attr( Sensei_Utils::get_current_url() ); ?>" method="get">
-			<select name="course-orderby" class="orderby">
+			<select name="course-orderby" class="orderby" onchange="this.form.submit()">
```

Comment:
It seems preferable to separate concerns and not mix JS with PHP. I understand wanting to eliminate a small JS file, but we're essentially trading off future maintainability for a small performance gain.

Reply:
I'd argue this as a maintainability gain, with not having to go through the "where is this piece of code applied" and "where is this behavior defined" hoops. Of course in most cases the scripts are larger and inlining them like this is not suitable, so I understand if we don't want to make exceptions and go for consistency.